atlp-rwanda/e-comm-team-techsmith-bn
# #184752075 Configure Postgres DB and Sequelize ORM

## What does this PR do?
Configure The App to use PostgreSQL as DB Engine and Sequelize ORM

## Description of Task to be completed?
1. Setup Postgres
2. Setup Sequelize ORM
3. Tested DB Connection

## How should this be manually tested?

1. Clone the repo
2. Run `npm install`
3. Create a `.env` file in the root of the app and add all necessary environment variables.
Check `.env.example` to find required environment variables need to run the application.
4. Run `npm run migrate` to create table
5. Run `npm run seed` to create seeders
6. Use postman to test this route `localhost:3000/api/users`. where 3000 is a  port number which can modified accordingly
 
## Any background context you want to provide?
The app comes with support for MONGODB, a NoSQL Database. PostgreSQL is an Open Source, fully featured and optimized SQL database engine for production ready environments.

## What are the relevant pivotal tracker/Trello stories?
1. [#184752075](https://www.pivotaltracker.com/n/projects/2635056/stories/184752075)
